starfish/imagestack/parser/tileset/_parser.py

```diff
@@ -99,9 +102,11 @@ def _load(self):
         self._expectations.report_dtype(self._r, self._ch, self._zplane, self._numpy_array.dtype)
 
     @property
-    def tile_shape(self) -> Tuple[int, int]:
+    def tile_shape(self) -> Mapping[Axes, int]:
         self._load()
-        tile_shape = self._numpy_array.shape
+        raw_tile_shape = self._numpy_array.shape
+        assert len(raw_tile_shape) == 2
```

Comment:
why are we starting to assert the length here? Is there a case when the raw shape wouldn't be length 2?

Reply:
If we start supporting N-dimensional chunks where N>2, then we may need to adjust the code.